Neos.Flow/Classes/Validation/Validator/AbstractValidator.php

```diff
@@ -90,6 +96,34 @@ function ($value) {
             ),
             $options
         );
+
+        $this->resultStack = [];
+    }
+
+    /**
+     * Push a new Result onto the Result stack and return it in order to fix cyclic calls to a single validator.
+     * @since Flow 5.1
+     * @see https://github.com/neos/flow-development-collection/pull/1275#issuecomment-414052031
+     * @return ErrorResult
+     */
+    protected function pushResult()
+    {
+        if ($this->result !== null) {
+            \array_push($this->resultStack, $this->result);
```

Comment:
This is namespaced, but usually we don' do this, and other places in this PR also don't do it.